tests/fuzz/CMakeLists.txt

```diff
@@ -26,6 +26,67 @@
 # STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF
 # THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+# Retrieve FuzzedDataProvider
+set(FDP_ROOT ${CMAKE_BINARY_DIR}/_deps/fuzzed_headers)
+set(FDP_DIR ${FDP_ROOT}/fuzzer)
+set(FDP_PATH ${FDP_DIR}/FuzzedDataProvider.h)
+file(MAKE_DIRECTORY ${FDP_DIR})
+if(NOT EXISTS ${FDP_PATH})
+  message(STATUS "Downloading FuzzedDataProvider.h to ${FDP_PATH}")
+  file(DOWNLOAD
+    https://raw.githubusercontent.com/llvm/llvm-project/main/compiler-rt/include/fuzzer/FuzzedDataProvider.h
```

Comment:
Why do you download this file during CMake step, instead of just having it in the source tree?

Reply:
That file is barely updated once a year, judging by its history. If you download the newest version, one day you might get a compilation error because of some incompatibility with the new version and you will have to fix the code anyway.

Reply:
OK.